REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6
 MathProgBase 0.6 0.8
-ReverseDiffSparse 0.8 0.9
-ForwardDiff 0.5 0.8
```

Comment:
Same, please keep the upper bound (0.9 would be fine since 0.8 works). ForwardDiff may break its API between releases.